packages/contracts/package.json

```diff
@@ -27,8 +27,9 @@
         "test:circleci": "yarn test:coverage"
     },
     "config": {
-        "abis": "src/artifacts/@(DummyToken|TokenTransferProxy|Exchange|TokenRegistry|MultiSigWallet|MultiSigWalletWithTimeLock|MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress|TokenRegistry|ZRXToken).json",
-        "contracts": "Exchange,DummyToken,ZRXToken,Token,WETH9,TokenTransferProxy,MultiSigWallet,MultiSigWalletWithTimeLock,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,MaliciousToken,TokenRegistry"
+        "abis": "src/artifacts/@(DummyToken|TokenTransferProxy|Exchange|TokenRegistry|MultiSigWallet|MultiSigWalletWithTimeLock|MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress|TokenRegistry|ZRXToken|AssetTransferProxy|ERC20TransferProxy_v1|ERC20TransferProxy|ERC721TransferProxy|DummyERC721Token).json",
+        "contracts": "Exchange,DummyToken,ZRXToken,Token,WETH9,TokenTransferProxy,MultiSigWallet,MultiSigWalletWithTimeLock,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,MaliciousToken,TokenRegistry,AssetTransferProxy,ERC20TransferProxy_v1,ERC20TransferProxy,ERC721TransferProxy,DummyERC721Token",
+        "dirs": "src/contracts,zeppelin:../../node_modules/zeppelin-solidity"
```

Comment:
What do we use from zeppelin?

Truffle by default adds just `node_modules` as a dir. This way you need to `import "zeppeling-solidity/something/something.sol" and it is clear what is from zeppelin.

Reply:
The ERC721 contracts come from Zeppelin. The library is quite extensive, so we may find it useful in other ways as well.

The updates I made to the deployer allow the directories to be namespaced. In this case, `0x` is in the global namespace and `zeppelin` is in its own namespace. If you check out the ERC721 Transfer Proxy, you'll see that we import contracts from zeppelin like this: `import "/zeppelin/contracts/token/ERC721/ERC721Token.sol";`